fury/actors/effect_manager.py

```diff
@@ -0,0 +1,431 @@
+
+
+WRAP_MODE_DIC = {"clamptoedge" : Texture.ClampToEdge,
+                 "repeat" : Texture.Repeat,
+                 "mirroredrepeat" : Texture.MirroredRepeat,
+                 "clamptoborder" : Texture.ClampToBorder}
+
+BLENDING_MODE_DIC = {"none" : 0, "replace" : 1,
+                     "modulate" : 2, "add" : 3,
+                     "addsigned" : 4, "interpolate" : 5,
+                     "subtract" : 6}
+
+def window_to_texture(
+        window : RenderWindow,
+        texture_name : str,
+        target_actor : Actor,
+        blending_mode : str = "None",
+        wrap_mode : str = "ClampToBorder",
+        border_color : tuple = (
+            0.0,
+            0.0,
+            0.0,
+            1.0),
+        interpolate : bool = True,
+        d_type : str = "rgb"):
+    """Capture a rendered window and pass it as a texture to the given actor.
+    Parameters
+    ----------
+    window : window.RenderWindow
+        Window to be captured.
+    texture_name : str
+        Name of the texture to be passed to the actor.
+    target_actor : Actor
+        Target actor to receive the texture.
+    blending_mode : str, optional
+        Texture blending mode. The options are:
+        1. None
+        2. Replace
+        3. Modulate
+        4. Add
+        5. AddSigned
+        6. Interpolate
+        7. Subtract
+    wrap_mode : str, optional
+        Texture wrapping mode. The options are:
+        1. ClampToEdge
+        2. Repeat
+        3. MirroredRepeat
+        4. ClampToBorder
+    border_color : tuple (4, ), optional
+        Texture RGBA border color.
+    interpolate : bool, optional
+        Texture interpolation.
+    d_type : str, optional
+        Texture pixel type, "rgb" or "rgba". Default is "rgb" 
+    """
+
+    windowToImageFilter = WindowToImageFilter()
+    windowToImageFilter.SetInput(window)
+    type_dic = {"rgb" : windowToImageFilter.SetInputBufferTypeToRGB, 
+                "rgba" : windowToImageFilter.SetInputBufferTypeToRGBA,
+                "zbuffer" : windowToImageFilter.SetInputBufferTypeToZBuffer}
+    type_dic[d_type.lower()]()
+    windowToImageFilter.Update()
+
+    texture = Texture()
+    texture.SetMipmap(True)
+    texture.SetInputConnection(windowToImageFilter.GetOutputPort())
+    texture.SetBorderColor(*border_color)
+    texture.SetWrap(WRAP_MODE_DIC[wrap_mode.lower()])
+    texture.SetInterpolate(interpolate)
+    texture.MipmapOn()
+    texture.SetBlendingMode(BLENDING_MODE_DIC[blending_mode.lower()])
+
+    target_actor.GetProperty().SetTexture(texture_name, texture)
+
+
+def texture_to_actor(
+        path_to_texture : str,
+        texture_name : str,
+        target_actor : Actor,
+        blending_mode : str = "None",
+        wrap_mode : str = "ClampToBorder",
+        border_color : tuple = (
+            0.0,
+            0.0,
+            0.0,
+            1.0),
+        interpolate : bool = True):
+    """Pass an imported texture to an actor.
+    Parameters
```

Comment:
same as before, break a line between the first docstring line and the Parameters